host/sgx/report.c

```diff
@@ -130,6 +133,7 @@ oe_result_t oe_get_report_v2(
     if (!enclave || !report_buffer || !report_buffer_size)
         OE_RAISE(OE_INVALID_PARAMETER);
 
+#ifndef OE_BUILD_HOST_VERIFY
```

Comment:
Host_verify package never call oe_get_report*(). We can remove the entire function if possible.

Reply:
host/sgx/report.c now has oe_get_report_v2(), moved rest to  host/sgx/report_common.c.